classes/Cart.php

```diff
@@ -4320,8 +4320,12 @@ public function deleteCustomizationToProduct($id_product, $index)
             AND `in_cart` = 0'
         );
 
+        if (!$cust_data) {
+            return $result;
```

Comment:
```suggestion
            return true;
```
and move the variable $result after the if statement (somewhere in line ~4326)
It is practically the same, except we don't init useless variable in case the $cust_data is falsy.

Reply:
done